stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,1011 @@
+            </div>
+            <div class="fd-dynamic-page__collapsible-header-visibility-container fd-dynamic-page__collapsible-header-visibility-container--no-shadow" aria-hidden="true">
+                <div class="fd-dynamic-page__collapsible-header-visibility-container--left-gradient"></div>
+                <div role="toolbar" aria-label="Header actions" class="fd-dynamic-page__collapsible-header-visibility-container--button-group">
+                    <button class="fd-dynamic-page__collapse-button fd-button fd-button--compact" 
+                            aria-label="See More" 
+                            aria-expanded="false" 
+                            aria-haspopup="true" 
+                            aria-controls="fddplhvcm2">
+                            <i class="sap-icon--slim-arrow-down"></i>
+                    </button>
+                </div>
+                <div class="fd-dynamic-page__collapsible-header-visibility-container--right-gradient"></div>
+            </div>
+            <div class="fd-dynamic-page__tabs--overflow">
+                <ul class="fd-tabs fd-tabs--l fd-dynamic-page__tabs fd-dynamic-page__tabs--sm fd-dynamic-page__tabs--add-shadow" role="tablist">
+                    <li role="listitem" class="fd-tabs__item">
+                        <a class="fd-tabs__link" aria-controls="fuCwV554" href="#fuCwV554" role="tab">
+                            <span class="fd-tabs__tag">
+                                Link
+                            </span>
+                        </a>
+                    </li>
+                    <li role="listitem" class="fd-tabs__item">
+                        <a class="fd-tabs__link" aria-controls="AiWfz169" aria-selected="true" href="#AiWfz169" role="tab">
+                            <span class="fd-tabs__tag">
+                                Selected
+                            </span>
+                        </a>
+                    </li>
+                </ul>  
+            </div>
+        </header>     
+        <div class="fd-dynamic-page__content fd-dynamic-page__content--sm">
+            <div class="fd-tabs__panel" aria-expanded="false" id="fuCwV554" role="tabpanel">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam 
+                eum tempore aliquid dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam eum tempore aliquid 
+                dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+            </div>
+            <div class="fd-tabs__panel" aria-expanded="true" id="AiWfz169" role="tabpanel">
+                <h4>This is an example of a summary line, without the expand/collapse feature.</h4>
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

> Avoid skipping heading levels: always start from `<h1>`, next use `<h2>` and so on.

```suggestion
                <h2 class="fd-title fd-title--h6 fd-title--wrap"><b>This is an example of a summary line, without the expand/collapse feature.</b></h2>
```

Reply:
should this also be done at line 680 `<h3>1:  Header with both collapse and pin buttons, expanded by default. </h3>` or since it already has h3, that would be sufficient?

Reply:
Those should also be `<h2>`s 👍

Reply:
done, please check.